drivers/sensor/pmw3360/pmw3360.c

```diff
@@ -617,8 +618,23 @@ static int pmw3360_async_init_fw_load_verify(struct pmw3360_data *dev_data)
 
 static void irq_handler(struct device *gpiob, struct gpio_callback *cb,
 			u32_t pins)
+{
+	int err;
+
+	err = gpio_pin_disable_callback(pmw3360_data.irq_gpio_dev,
+					PMW3360_IRQ_GPIO_PIN);
+	if (unlikely(err)) {
+		LOG_ERR("Cannot disable IRQ");
+		k_panic();
+	}
+
+	k_work_submit(&pmw3360_data.handler_trigger_work);
+}
+
+static void handler_trigger(struct k_work *work)
```

Comment:
`trigger_handler` seems to be a more adequate name (same goes for `trigger_handler_work` as the associated field in the device data).

Reply:
it is "a trigger of a handler" not "a handler of a trigger" :)

Reply:
Fixed